packages/basti/src/cli/config/get-command-input.ts

```diff
@@ -64,14 +66,25 @@ export function getConnectCommandInputFromConfig(
           rdsClusterId: connectionTarget.rdsCluster,
           awsClientConfig,
         }
-      : isElasticacheClusterTargetConfig(connectionTarget)
+      : isElasticacheRedisClusterTargetConfig(connectionTarget)
       ? {
-          elasticacheClusterId: connectionTarget.elasticacheCluster,
+          elasticacheRedisClusterId: connectionTarget.elasticacheRedisCluster,
           awsClientConfig,
         }
-      : isElasticacheNodeTargetConfig(connectionTarget)
+      : isElasticacheRedisNodeTargetConfig(connectionTarget)
       ? {
-          elasticacheNodeId: connectionTarget.elasticacheNode,
+          elasticacheRedisNodeId: connectionTarget.elasticacheRedisNode,
+          awsClientConfig,
+        }
+      : isElasticacheMemcachedClusterTargetConfig(connectionTarget)
+      ? {
+          elasticacheRedisClusterId:
```

Comment:
Shouldn't it be `elasticacheMemcachedClusterId` here and `elasticacheMemcachedNodeId` in the next case?